dunwoody_disaster/animations/basic_attack.py

```diff
@@ -1,23 +1,29 @@
 import pygame
 from dunwoody_disaster.animations.PygameAnimation import PygameAnimation
-import dunwoody_disaster as DD
-from typing import override
-from dunwoody_disaster.CharacterFactory import CharacterFactory
-from dunwoody_disaster import Item
+
+# from typing import override
 from PySide6.QtCore import Signal
 
 
 class AttackAnimation(PygameAnimation):
-    def __init__(self, background: str, player: str, enemy: str, weapon: str, onFinish: Signal):
+    def __init__(
+        self, background: str, player: str, enemy: str, weapon: str, onFinish: Signal
+    ):
         super().__init__()
         self.bkg = pygame.image.load(background).convert_alpha()  # 666x360 image
-        self.player = pygame.transform.scale(pygame.image.load(player), (100, 100)).convert_alpha()
-        self.enemy = pygame.transform.scale(pygame.image.load(enemy), (100, 100)).convert_alpha()
-        self.weapon = pygame.transform.scale(pygame.image.load(weapon), (100, 100)).convert_alpha()
+        self.player = pygame.transform.scale(
+            pygame.image.load(player), (100, 100)
+        ).convert_alpha()
+        self.enemy = pygame.transform.scale(
+            pygame.image.load(enemy), (100, 100)
+        ).convert_alpha()
+        self.weapon = pygame.transform.scale(
+            pygame.image.load(weapon), (100, 100)
+        ).convert_alpha()
         self.weapon_x = 60
         self.finished = onFinish
 
-    @override
+    # @override
```

Comment:
pt 5

Reply:
I'll resolve these comments, because I fixed these overrides in another branch.